crates/tabby-common/src/api/structured_doc.rs

```diff
@@ -20,6 +20,7 @@ pub struct DocSearchHit {
 pub enum DocSearchDocument {
     Web(DocSearchWebDocument),
     Issue(DocSearchIssueDocument),
+    Pull(DocSearchPullRequest),
```

Comment:
```suggestion
    PullRequest(DocSearchPullRequest),
```

Please also update other naming /variables to reflect the change.